data/cameras.xml

```diff
@@ -681,7 +711,7 @@
 			<Horizontal y="4" height="44"/>
 		</BlackAreas>
 	</Camera>
-	<Camera make="Canon" model="Canon EOS M2">
```

Comment:
Yeah, we had a bunch of these from cameras that originally where ="no". I've removed them all now as it's just noise.